index.html

```diff
@@ -1,12 +1,259 @@
+            src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2645.5055203692987!2d35.05302421554022!3d48.46601637925102!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x9ffb0c281aeab08e!2z0JrQvtC80L_RjNGO0YLQtdGA0L3QsNGPINGI0LrQvtC70LAgSGlsbGVs!5e0!3m2!1sru!2sua!4v1606076668933!5m2!1sru!2sua"
+            width="600"
+            height="450"
+            style="border: 0"
+            allowfullscreen=""
+            aria-hidden="false"
+            tabindex="0"
+          ></iframe>
+        </figure>
+      </section>
+      <section class="section_1">
+        <h2>Contact</h2>
+        <p class="stay_text">
+          Questions or concerns? Just fill out the form below and our support
+          team will get back to you within 24 hours. But I must explain to you
+          how all this mistaken idea of denouncing pleasure and praising pain
+          was born and I will give you a complete account of the system, and
+          expound the actual teachings of the great explorer of the truth, the
+          master-builder of human happiness.
+        </p>
+        <form>
+          <label
+            ><input
+              type="text"
+              name="user_first_name"
+              placeholder="First Name"
+              required
+          /></label>
+          <label
+            ><input
+              type="text"
+              name="user_last_name"
+              placeholder="Last Name"
+              required
+          /></label>
+          <label
+            ><input
+              type="tel"
+              name="user_phone_number"
+              placeholder="Phone Number"
+              required
+          /></label>
+          <label>
+            <select name="select_service" class="select">
+              <option disabled selected>Select Service</option>
+              <option value="Back-end">Back-end</option>
+              <option value="Design">Design</option>
+            </select>
+          </label>
+          <button type="submit" class="form_button">SUBMIT NOW</button>
+        </form>
+      </section>
+    </main>
+    <footer class="footer">
+      <div>
+        <h3>Address</h3>
+        <p class="adress_text">
+          Pipang Ltd, Griva Digeni,<br />
+          81-83 Jacovides Tower, 1st Floor <br />
+          1090 Picosia USA
+        </p>
+      </div>
+      <div>
+        <h3>Services</h3>
+        <ul class="ul_services">
+          <li><a href="#" class="li_serv">Overview</a></li>
+          <li><a href="#" class="li_serv">Features</a></li>
+          <li><a href="#" class="li_serv">Technology</a></li>
+          <li><a href="#" class="li_serv">Terms & Conditions</a></li>
+          <li><a href="#" class="li_serv">Privecy</a></li>
+        </ul>
+      </div>
+      <div>
+        <h3>Get in Touch</h3>
+        <a href="mailto:info@cloudbudget.com" class="tel_mail"
+          >info@cloudbudget.com</a
+        >
+        <a href="tel:+1844-721-7120" class="tel_mail">+1 844-721-7120</a>
```

Comment:
href attribute should not contain dashes. remove

Reply:
Ok